presto-docs/src/main/sphinx/functions/noisy.rst

```diff
@@ -123,6 +123,29 @@ is supported via the Sketch-Flip-Merge (SFM) data sketch [Hehir2023]_.
         Unlike :func:`approx_set`, this function returns ``NULL`` when ``col`` is empty.
         If this behavior is undesirable, use :func:`coalesce` with :func:`noisy_empty_approx_set_sfm`.
 
+.. function:: noisy_approx_set_sfm_from_index_and_zeros(col_index, col_zeros, epsilon, buckets[, precision]) -> SfmSketch
+
+    Returns an SFM sketch of the input values in ``col_index`` and ``col_zeros``. This is similar to
+    :func:`noisy_approx_set_sfm` except that function calculates a ``Murmur3Hash128.hash64()`` of ``col``,
+    and calculates the SFM PCSA bucket index and number of trailing zeros as described in [FlajoletMartin1985]_,
+    whereas in this function the caller must explicitly calculate the hash bucket index
```

Comment:
```suggestion
    In this function, the caller must explicitly calculate the hash bucket index
```
Nit: suggest breaking this sentence into two shorter sentences to improve readability.